opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -0,0 +1,337 @@
+		as well as error handling. 
+		"""
+		if view_datas:
+			self.upload_stats(view_datas)
+		
+	def upload_stats(self, view_datas):
+		""" It receives an array of view_data object 
+			and create time series for each value
+		"""
+		for request in self.make_request(view_datas, 
+							MAX_TIME_SERIES_PER_UPLOAD):
+			self.client.create_time_series(request[CONS_NAME],
+											request[CONS_TIME_SERIES])
+
+	def make_request(self, view_datas, limit):
+		""" Create the data structure that will be 
+			sent to Stackdriver Monitoring
+		"""
+		requests = []
+		time_series = []
+
+		resource = self.options.resource
+		for view_data in view_datas:
+			series = monitoring_v3.types.TimeSeries()
+			series.metric.type = namespaced_view_name(view_data.view.name)
+
+			if not resource:
+    				series.resource.type = 'global'
+			else:
+				series.resource = resource
+
+			for tag_value, agg in view_data.tag_value_aggregation_map.items():
+				point = series.points.add()
+				if agg.aggregation_type is aggregation.Type.DISTRIBUTION:
+					agg_data = view_data.tag_value_aggregation_map.get(tag_value).aggregation_data
+					
+					point.value.distribution_value.count = agg_data.count_data
+					point.value.distribution_value.mean = agg_data.mean_data
+					point.value.distribution_value.sum_of_squared_deviation = agg_data.sum_of_sqd_deviations
+
+					# Uncomment this when stackdriver supports Range
+					# point.value.distribution_value.range.min = agg_data.min
+					# point.value.distribution_value.range.max = agg_data.max
+
+					point.value.distribution_value.bucket_options.explicit_buckets.bounds.extend(list(map(float,agg_data.bounds)))
+					point.value.distribution_value.bucket_counts.extend(list(map(int,agg_data.counts_per_bucket)))
+				elif type(tag_value.value) is int:
+					point.value.int64_value = int(tag_value.value)
+				elif type(tag_value.value) is float:
+					point.value.float_value = float(tag_value.value)
+				elif type(tag_value.value) is str:
+					point.value.string_value = str(tag_value.value)
+
+				start = datetime.datetime.strptime(view_data.start_time,"%Y-%m-%dT%H:%M:%S.%fZ")
+				end = datetime.datetime.strptime(view_data.end_time,"%Y-%m-%dT%H:%M:%S.%fZ")
+
+				timestamp_start = (start - datetime.datetime(1970, 1, 1)).total_seconds()
+				timestamp_end = (end - datetime.datetime(1970, 1, 1)).total_seconds()
+
+				point.interval.end_time.seconds = int(timestamp_end)
+				point.interval.end_time.nanos = int((timestamp_end - point.interval.end_time.seconds) * 10**9)
+
+				#
+				# Uncomment this when LastValue gets supported
+				#
+				#if not agg.aggregation_type is aggregation.Type.LASTVALUE:
+				if timestamp_start == timestamp_end: # avoiding start_time and end_time to be equal
+					timestamp_start = timestamp_start - 1
+				point.interval.start_time.seconds = int(timestamp_start)
+				point.interval.start_time.nanos = int((timestamp_start - point.interval.end_time.seconds) * 10**9)
```

Comment:
This looks wrong and will result in a negative number, as `point.interval.end_time.seconds` will ALWAYS be greater than `timestamp_start`. Perhaps you meant to say
```python
ns = int((timestart - point.interval.start_time.seconds) * 1e9)
```

Reply:
I think that you are correct, however I did not found any example using the start() from view_data, because only start() and end() will create two different timestamps. If the user do not use them, start_time and end_time will be the same value. I'm going to fix this, thank you.